master/pkg/schemas/expconf/experiment_config.go

```diff
@@ -117,6 +121,9 @@ func (l LabelsV0) MarshalJSON() ([]byte, error) {
 
 // UnmarshalJSON implements the json.Unmarshaler interface.
 func (l *LabelsV0) UnmarshalJSON(data []byte) error {
```

Comment:
remind me, why did/do we support a list and map here?

Reply:
I have no idea.  I know it happened on commit `b8c5cdb72b` but I can't figure out why that was what we did.

Reply:
Weird. I wonder why it wasn't _always_ a list.